<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Grapher</class>
 <widget class="QWidget" name="Grapher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QMdiArea" name="mdiArea">
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="graphertoolButton">
          <property name="text">
           <string>Graph</string>
          </property>
          <property name="icon">
           <iconset resource="vTrazer.qrc">
            <normaloff>:/img/results</normaloff>:/img/results</iconset>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="graphList"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionCascade">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vTrazer.qrc">
     <normaloff>:/img/wcascade</normaloff>:/img/wcascade</iconset>
   </property>
   <property name="text">
    <string>Cascade Arrange</string>
   </property>
   <property name="toolTip">
    <string>Cascade Arrange Graph</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vTrazer.qrc">
     <normaloff>:/img/wtile</normaloff>:/img/wtile</iconset>
   </property>
   <property name="text">
    <string>Tile Arrange</string>
   </property>
   <property name="toolTip">
    <string>Tile arrange Graph</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="vTrazer.qrc">
     <normaloff>:/img/graph</normaloff>:/img/graph</iconset>
   </property>
   <property name="text">
    <string>New Graph</string>
   </property>
   <property name="toolTip">
    <string>New Graph</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="vTrazer.qrc"/>
 </resources>
 <connections/>
</ui>
